languages/messages/MessagesQqq.php

```diff
@@ -3772,4 +3772,6 @@
 
 {{Identical|Other}}',
 
+'uploadnologintext' => 'Information that user have to log in to upload files or add videos. Displayed for non logged in users that perform upload action. 1st parameter is LogIn page link.',
```

Comment:
Should probably be "Information that the user has to log in to upload files or add videos. Displayed for non-logged in users that perform upload action. 1st parameter is LogIn page link." It's slightly more grammatically correct.